alembic/versions/2024_02_19_43eb680d6181_add_order_case_table.py

```diff
@@ -0,0 +1,33 @@
+"""Add order case table
+
+Revision ID: 43eb680d6181
+Revises: d241d8c493fb
+Create Date: 2024-02-19 10:13:21.075891
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "43eb680d6181"
+down_revision = "d241d8c493fb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "order_case",
+        sa.Column("id", sa.Integer, nullable=False),
+        sa.Column("order_id", sa.Integer, nullable=False, index=True),
+        sa.Column("case_id", sa.Integer, nullable=False, index=True),
+        sa.ForeignKeyConstraint(["order_id"], ["order.id"], ondelete="CASCADE"),
+        sa.ForeignKeyConstraint(["case_id"], ["case.id"], ondelete="CASCADE"),
```

Comment:
Eh, I'm not super familiar with the CASCADE option but I would not like an order to be deleted if one of the cases are 😅 Are you sure "CASCADE" should be set on both these lines?

Reply:
As I understand it, this would delete the link entry, and we would be blocked from deleting both orders and cases with entries in this table without it. I just modeled it after the CaseSample table though, should probably be checked.

Reply:
Checked!